CHANGELOG.md

```diff
@@ -1,3 +1,22 @@
+## v1.2.1
+
+ENHANCEMENTS:
+
+- `data-source/auth0_organization`: Refactor organization-related pagination method from `'Offset pagination'` to `'Checkpoint pagination'` ([#965](https://github.com/auth0/terraform-provider-auth0/pull/965))
```

Comment:
Maybe we can just say "Switch from offset pagination to checkpoint pagination when listing organizations/members" to remove the 1000 result limit